package.json

```
@@ -1,18 +1,21 @@
 {
   "name": "@holo-host/web-sdk",
-  "version": "0.5.3",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.6.0-prerelease",
```

@JettTech - based on a conversation in US standup today, we agreed on this version number. 0.6.0 is still considered a breaking change in semver, and adding the `-prerelease` extra safeguards from external devs updating.